BriskScript/Sources/Brisk/String.swift

```diff
@@ -113,6 +113,19 @@ extension String {
         self = self.replacingOccurrences(of: regex, with: replacement, options: options.union([.regularExpression]))
     }
 
+	func getCapturedGroupsFrom(regexPattern: String)-> [String]? {
```

Comment:
Do you think this might work better as `captureGroups(from regexPattern: String)`? It's shorter and straight to the point. (Also, watch out for the space before `->`!)

Reply:
I totally agree `captureGroups(from regexPattern: String)` is a much better name! With space too!  👍🏼 

```
func captureGroups(from regexPattern: String) -> [String]? {
		let text = self
		let regex = try? NSRegularExpression(pattern: regexPattern)
		let match = regex?.firstMatch(in: text, range: NSRange(text.startIndex..., in: text))

		if let match = match {
			return (0..<match.numberOfRanges).compactMap {
				$0 > 0 ? String(text[Range(match.range(at: $0), in: text)!]) : nil
			}
		}
		return nil
	}

```

Reply:
Great! Did you already push some changes that I missed, or are you going to do that so I can merge?